resources/views/projects/details.blade.php

```diff
@@ -90,7 +90,74 @@
 					@endforeach
 				</div>
 			</div>
+			<!-- Progress section -->
+			<div class="sixteen wide column">
+				<div class="ui left aligned detail-container">
+					<p><strong>Progress</strong></p>
+					<div class="ui blue progress">
+						<div class="bar">
+							<div class="progress"></div>
+						</div>
+						<div class="label">Currently on: Design</div>
+					</div>
+					<div class="ui right floated button" onclick="displayMilestones()">
+						<a class="milestones" title="See project's milestones">See milestones</a>
+					</div>
+					<div class="ui list milestone-section hidden">
+						<div class="item done">
+							<i class="large circle green icon"></i>
+							<div class="content">
+								<div class="header">Ideation</div>
+								<div class="description">Finished on 4/20</div>
+							</div>
+						</div>
+						<div class="item current">
+							<i class="large circle blue icon"></i>
+							<div class="content">
+								<div class="header">Design</div>
+								<div class="description">Estimated date: 4/25</div>
+							</div>
+						</div>
+						<div class="item coming-up">
+							<i class="large circle grey icon"></i>
+							<div class="content">
+								<div class="header">Planning</div>
+								<div class="description">Estimated date: 4/30</div>
+							</div>
+						</div>
+						<div class="item coming-up">
+							<i class="large circle grey icon"></i>
+							<div class="content">
+								<div class="header">Execution</div>
+								<div class="description">Estimated date: 6/10</div>
+							</div>
+						</div>
+						<div class="item coming-up">
+							<i class="large circle grey icon"></i>
+							<div class="content">
+								<div class="header">Test</div>
+								<div class="description">Estimated date: 6/30</div>
+							</div>
+						</div>
+						<a class="ui right floated primary button" title="Edit milestone map" href="">Edit milestone map</a>
```

Comment:
Todos los botones de Create/Update milestones deberían estar filtrados específicamente para los project collaborators. Ya están trabajando en los métodos aquí: #182 so let's not forget to add those once they are merged. :) 

Igualmente, por lo general sólo cargo las partes del script que son relevantes para el usuario que está en el view. 

Es decir:

```php
@if($project->IsProjectStakeholder)
// your code here
@endif
```

Reply:
Si! Lo dejé pendiente porque vi que estaban definiendo esos detalles. Thank you!